Common/src/main/java/com/blamejared/crafttweaker/impl/script/scriptrun/ScriptRunLogger.java

```diff
@@ -58,13 +57,13 @@ public void error(final String message) {
     @Override
     public void throwingErr(final String message, final Throwable throwable) {
         
-        CraftTweakerAPI.LOGGER.throwing(Level.ERROR, throwable);
+        CraftTweakerAPI.LOGGER.error("Error while running scripts:", throwable);
     }
     
     @Override
     public void throwingWarn(final String message, final Throwable throwable) {
-        
-        CraftTweakerAPI.LOGGER.throwing(Level.WARN, throwable);
+    
+        CraftTweakerAPI.LOGGER.warn("Warning while running scripts:", throwable);
```

Comment:
I feel like the wording here is slightly off, but I can't pinpoint why

Reply:
Yea, I wasn't a fan of it either, the issue is, if we make it just:
```
        CraftTweakerAPI.LOGGER.throwing(Level.WARN, throwable);
```

It displays as:
```
throwing Cannot parse X
```
and I couldn't figure out how to get it to have just the throwable with no text